### ISSUE TITLE
Question: Inconsistent numerical ranges between mask/control conditioning and content channels in A2 model

### ISSUE THREAD

**Issue by veyorokon** [SkyworkAI/SkyReels-A2#35]
Observation

  While implementing training code for SkyReels-A2, I noticed that the mask channel and content channels operate in different numerical ranges:

  - Content latents operate in the standard diffusion model [-1,1] range
  - Mask channel uses binary values [0,1] range (in pipeline_a2.py)

 From pipeline_a2.py
  mask_lat_size = torch.ones(batch_size, 1, num_frames, latent_height, latent_width)
  mask_lat_size[:, :, list(range(1, num_frames))] = 0
Later concatenated directly with content latents

  Questions

  1. Was this difference in ranges an intentional design choice?
  2. Did you observe any training advantages/disadvantages to keeping the mask in [0,1] rather than normalizing it to [-1,1]?
  3. Would you recommend maintaining this approach for implementations, or normalizing everything to the same range?

  This isn't causing functional issues in our implementation, but I'm curious about the mathematical reasoning behind this design decision,
   as diffusion models typically prefer consistent normalization across all channels.

  Thank you for creating and sharing this excellent model architecture!
